server/routes/createGoal/createGoalsController.ts

```diff
@@ -12,44 +12,37 @@ export default class CreateGoalsController {
     const { prisonNumber } = req.params
     const { prisonerSummary } = req.session
 
-    req.session.createGoalsForm = req.session.createGoalsForm || []
```

Comment:
This needed quite a bit of refactoring as Adam had copied most of it from the current create goal (singular) controller which works in quite a different way (it uses arrays of forms on the session)